dotcom-rendering/src/components/CardHeadline.tsx

```diff
@@ -43,77 +46,151 @@ type Props = {
 	isHighlights?: boolean;
 };
 
-const fontStyles = ({ size }: { size: SmallHeadlineSize }) => {
+/** These represent a new set of fonts. They are extra large font sizes that, as a group, are only used on headlines */
+const headlineFontStyles = ({ size }: { size: SmallHeadlineSize }) => {
 	switch (size) {
+		// we don't have a ginormous size in designs. For now this defaults to huge.
 		case 'ginormous':
-			return css`
-				${from.desktop} {
-					${headlineMedium50}
-				}
-			`;
 		case 'huge':
-			return css`
-				${headlineMedium28}
+			// this needs to be updated to font size 64 once this is available in source.
+			return `
+				${headlineMedium50}
+				font-size: 64px;
 			`;
+
 		case 'large':
-			return css`
-				${headlineMedium24}
-			`;
+			return `${headlineMedium50}`;
+
 		case 'medium':
-			return css`
-				${headlineMedium20}
-			`;
+			return `${headlineMedium42}`;
+
 		case 'small':
-			return css`
-				${headlineMedium17}
-			`;
+			return `${headlineMedium34}`;
+
 		case 'tiny':
-			return css`
-				${headlineMedium14}
-			`;
+			return `${headlineMedium28}`;
 	}
 };
 
-const fontStylesOnMobile = ({ size }: { size: SmallHeadlineSize }) => {
-	switch (size) {
-		case 'ginormous':
-			return css`
-				${until.mobileLandscape} {
-					${headlineMedium34}
-				}
-				${between.mobileLandscape.and.desktop} {
-					${headlineMedium42}
-				}
-			`;
-		case 'huge':
-			return css`
-				${until.desktop} {
-					${headlineMedium24}
-				}
-			`;
-		case 'large':
-			return css`
-				${until.desktop} {
-					${headlineMedium20}
-				}
-			`;
-		case 'medium':
-			return css`
-				${until.desktop} {
-					${headlineMedium17}
-				}
-			`;
-		case 'small':
+const fontStyles = ({
+	size,
+	fontRange,
+}: {
+	size: SmallHeadlineSize;
+	fontRange: 'regular' | 'headline';
+}) => {
+	switch (fontRange) {
+		case 'headline':
+			return headlineFontStyles({ size });
+		case 'regular':
+		default:
+			switch (size) {
+				case 'ginormous':
+					return css`
+						${from.desktop} {
+							${headlineMedium50}
+						}
+					`;
+				case 'huge':
+					return css`
+						${headlineMedium28}
+					`;
+				case 'large':
+					return css`
+						${headlineMedium24}
+					`;
+				case 'medium':
+					return css`
+						${headlineMedium20}
+					`;
+				case 'small':
+					return css`
+						${headlineMedium17}
+					`;
+				case 'tiny':
+					return css`
+						${headlineMedium14}
+					`;
+			}
+	}
+};
+
+const fontStylesOnTablet = ({
+	size,
+	fontRange,
+}: {
+	size?: SmallHeadlineSize;
+	fontRange: 'regular' | 'headline';
+}) => {
+	if (!size) return;
+	if (fontRange === 'headline') {
```

Comment:
More of a style suggestion, I think?

```suggestion
	if (size && fontRange === 'headline') {
```